tests/dummy/app/components/my-stateful-component/component.js

```diff
@@ -14,22 +16,22 @@ export default Ember.Component.extend(Stateful, {
 
     dead: {
       _enter() {
```

Comment:
the dummy app needs to use `_default: true` key-value pairs in its action hashes instead of the `states` properties as and to rename all the `_enter`s to `_try`s and `_exit`s to `_finally`s

Reply:
yeah, forgot about the dummy app...